### ISSUE TITLE
Create installer

### ISSUE THREAD

**Issue by bastianschwarz** [bastianschwarz/warthog.dcs.f18#6]
It would be really handy to provide an Installer/Updater that extracts the target scripts, creates a shortcut to run TARGET + DCS and installs the DCS button mapping into the SavedGames folder.

This is probably the next step after #4 is finished.

Basics:

- [ ] Ask for DCS Folder(s) and TARGET folder in order to create the shortcut
- [ ] Ask for SavedGame folder in order to install lua.diff keybindings
- [ ] Select the installations that should be used (Stable + OpenBeta)

Nice to have:
- [ ] Backup/Restore of existing profiles
- [ ] Activate/Deactivate binds on Startup (based on Backup/Restore)
- [ ] Compile features by choosing the header files
- [ ] DCS Version/Compatiblity detection

Also: Defining an installer that quickyl grows in complexity is totally not feature creep ...
